air/src/utils.rs

```diff
@@ -1,8 +1,172 @@
 use alloc::vec::Vec;
-use core::ops::Range;
+use core::ops::{Add, AddAssign, Index, IndexMut, Mul, Range, Rem, Shr, Sub};
 
 use super::FieldElement;
-use vm_core::utils::range as create_range;
+use vm_core::{utils::range as create_range, Felt};
+
+// NEWTYPE WRAPPERS
+// ================================================================================================
+
+/// A newtype wrapper around a usize value representing a step in the execution trace.
+#[derive(Debug, Copy, Clone)]
+pub struct Step(u32);
```

Comment:
I would move the definition of `Step` to `air/src/trace/row.rs` (module name depends on outcome of previous comment about naming of `Step`)